services/provider/server/server.go

```diff
@@ -278,7 +278,7 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 		Name: monConfigMap,
 		Kind: "ConfigMap",
 		Data: mustMarshal(map[string]string{
-			"data":     cmData[0], // Address of first mon
+			"data":     strings.Join(cmData, ", "), // IP Address of all mon's
```

Comment:
Not sure rook can handle the space. Here's a sample from a test cluster. And is the mon name included with the `cmData`?
```
data: a=10.110.186.73:6789,b=10.105.227.50:6789,c=10.104.4.237:6789
```

```suggestion
			"data":     strings.Join(cmData, ","), // IP Address of all mon's
```